eng/Versions.props

```diff
@@ -97,6 +90,16 @@
     <VisualStudioLanguageServerProtocolVersion>17.1.11</VisualStudioLanguageServerProtocolVersion>
   </PropertyGroup>
   <PropertyGroup Label="Manual">
+    <!-- dotnet/runtime packages -->
```

Comment:
Could you describe the pros and cons of making these versions manual versus having a weekly or even monthly subscription on the dotnet/runtime 6.0 packages❔ If the idea is to avoid depending on unreleased bits from dotnet/runtime, you may want to go further and remove some feeds from NuGet.config.

Reply:
Definitely cool with pinning these!